control.py

```diff
@@ -17,69 +22,215 @@ def hello():
         return request.json
 
 
-@app.route("/diretores", methods=["GET", "POST", "DELETE"])
-def diretor():
+@app.route("/diretores", methods=["GET", "POST"])
+def diretor_GetPost():
     if request.method == "GET":
-        return jsonify(query("Select * From diretores"))
+        nome_completo = nome_usuario_from_web(**request.args)
+        diretores = select_diretores(nome_completo)
+        diretores_from_db = [diretor_from_db(diretor) for diretor in diretores]
+        return jsonify(diretores_from_db)
     elif request.method == "POST":
         diretor = diretor_from_web(**request.json)
         if valida_diretor(**diretor):
-            insert_diretor(**diretor)
-            diretor_novo = get_diretor(diretor['nome_completo'])
-            return jsonify(diretor_from_db(diretor_novo))
+            id_diretor = insert_diretor(**diretor)
+            diretor_new = get_diretor(id_diretor)
+            return jsonify(diretor_from_db(diretor_new))
         else:
-            return jsonify({"erro": "diretor invalido"})
-    elif request.method == "DELETE":
-        diretor = diretor_from_web_id(**request.json)
-        if valida_diretor_id(**diretor):
-            deletar_diretor(**diretor)
-            diretor_deletado = deletar_diretor(diretor("id"))
-            return jsonify(diretor_from_db_id(diretor_deletado))
-        else:
-            return jsonify({"Erro": "valor nao deletado"})
+            return jsonify({"Valor": "Nao adicionado a tabela"})
 
 
-@app.route("/diretores/<int:id>", methods=["PATCH"])
-def patch_diretor(id):
-    if request.method == "PATCH":
+@app.route("/diretores/<int:id>", methods=["PUT", "PATCH", "DELETE"])
+def diretor_delete_update(id):
+    if request.method == "DELETE":
+        try:
+            deletar_diretor(id)
+            return jsonify({"Valor": "Deletado"})
+        except:
+            return jsonify({"Valor": "Nao deletado"})
+    elif request.method == "PUT" or "PATCH":
         diretor = diretor_from_web(**request.json)
         if valida_diretor(**diretor):
             update_diretor(id, **diretor)
-            diretor_alterado = get_diretor(diretor(id))
+            diretor_alterado = get_diretor(id)
             return jsonify(diretor_from_db(diretor_alterado))
         else:
             return jsonify({"Erro": "Valor nao alterado"})
 
 
-@app.route("/usuarios", methods=["GET"])
-def usuarios_get():
-    nome_completo = nome_usuario_from_web(**request.args)
-    usuarios = select_usuarios(nome_completo)
-    usuarios_from_db = [usuario_from_db(usuario) for usuario in usuarios]
-    return jsonify(usuarios_from_db)
+@app.route("/usuarios", methods=["GET", "POST"])
+def usuario_getpost():
```

Comment:
Importante separar as palavras com _, e manter consistência na nomenclatura
```suggestion
def usuario_get_post():
```